docs: Edit migrating data with mirrormaker2 (#746)
redpanda-data/documentation: docs/platform/data-management/data-migration.mdx

        └── connect-mirror-maker.properties
```

> The sample configuration is a great place to understand a number of the settings for MirrorMaker 2.

Reviewer: ```suggestion
The sample configuration describes a number of the settings for MirrorMaker 2.
```
